### PR TITLE
ES Lint Changes

### PR DESCRIPTION
**Pull request by Sashank999**

### CODE REVIEW

File: extension/pages/popup/popup.js [hunk 1]
```diff
@@ -33,10 +33,10 @@ const initiatedPages = {};
 	}
 	document.find("#pages .right-nav li[to='settings']").addEventListener("click", () => chrome.runtime.openOptionsPage());
 
-	if (!settings.pages.popup.dashboard) document.find('#pages li[to="dashboard"]').classList.add("hidden");
-	if (!settings.pages.popup.marketSearch) document.find('#pages li[to="market"]').classList.add("hidden");
-	if (!settings.pages.popup.calculator) document.find('#pages li[to="calculator"]').classList.add("hidden");
-	if (!settings.pages.popup.stocksOverview) document.find('#pages li[to="stocks"]').classList.add("hidden");
+	if (!settings.pages.popup.dashboard) document.find("#pages li[to=\"dashboard\"]").classList.add("hidden");
```
Comment (DeKleineKobini) [inline extension/pages/popup/popup.js:36]: I don't agree with these changes. `'` over escaping imo

File: extension/scripts/background.js [hunk 2]
```diff
@@ -67,21 +67,28 @@ async function convertDatabase() {
...
 					let useCurrent = true;
 
-					if (defaultStorage[key].type === "array") {
-						if (!Array.isArray(oldStorage[key])) {
-							useDefault();
-							useCurrent = false;
-						}
-					} else if (
-						!defaultStorage[key].type.split("|").some((value) => value === typeof oldStorage[key] || (value === "empty" && oldStorage[key] === ""))
+					if (
```
Comment (DeKleineKobini) [inline extension/scripts/background.js:76]: I don't think I like this either. `useDefault` is so much easier to understand what it actually does. It makes the condition overcomplicated imo as well.

File: extension/scripts/features/abroad-people-filter/ttPeopleFilter.js [hunk 3]
```diff
@@ -261,7 +261,7 @@
 
 		function show() {
 			row.classList.remove("hidden");
-			delete row.dataset.hideReason;
```
Comment (DeKleineKobini) [inline extension/scripts/features/abroad-people-filter/ttPeopleFilter.js:264]: Why would you change this?

File: extension/scripts/features/align-left/ttAlignLeft.entry.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
 "use strict";
 
 (async () => {
+	await requireElement("body");
+	if (isFlying() || isAbroad()) return;
```
Comment (DeKleineKobini) [inline extension/scripts/features/align-left/ttAlignLeft.entry.js:5]: Why would you disable left align when flying?

File: extension/scripts/features/computer-link/ttComputerLink.js [hunk 5]
```diff
@@ -49,7 +49,7 @@
 							</svg>
 						</span>
 					</span>
-					${mobile ? "" : '<span id="pc">Computer</span>'}
+					${mobile ? "" : "<span id=\"pc\">Computer</span>"}
```
Comment (DeKleineKobini) [inline extension/scripts/features/computer-link/ttComputerLink.js:52]: I don't agree with these changes. `'` over escaping imo

File: extension/scripts/features/custom-links/ttCustomLinks.js [hunk 6]
```diff
@@ -1,6 +1,8 @@
 "use strict";
 
 (async () => {
+	if (isFlying() || isAbroad()) return;
```
Comment (DeKleineKobini) [inline extension/scripts/features/custom-links/ttCustomLinks.js:4]: Mobile does have the navbar, even when flying.

Comment (DeKleineKobini) [reply extension/scripts/features/custom-links/ttCustomLinks.js:4]: You should probably create a function to move that logic away.

File: extension/scripts/features/hide-areas/ttHideAreas.entry.js [hunk 7]
```diff
@@ -1,6 +1,9 @@
 "use strict";
 
 (async () => {
+	await requireElement("body");
+	if (isFlying() || isAbroad()) return;
```
Comment (DeKleineKobini) [inline extension/scripts/features/hide-areas/ttHideAreas.entry.js:5]: Mobile does have the navbar, even when flying.

File: extension/scripts/features/hide-icons/ttHideIcons.entry.js [hunk 8]
```diff
@@ -1,6 +1,9 @@
 "use strict";
 
 (async () => {
+	await requireElement("body");
+	if (isFlying() || isAbroad()) return;
```
Comment (DeKleineKobini) [inline extension/scripts/features/hide-icons/ttHideIcons.entry.js:5]: Mobile does have the navbar, even when flying.

File: extension/scripts/global/functions/containers.js [hunk 9]
```diff
@@ -62,7 +62,7 @@ function createContainer(title, options = {}) {
 				<div class="title ${collapsed ? "collapsed" : ""}">
 					<div class="text">${title}</div>
 					<div class="options"></div>
-					${options.collapsible ? '<i class="icon fas fa-caret-down"></i>' : ""}
+					${options.collapsible ? "<i class=\"icon fas fa-caret-down\"></i>" : ""}
```
Comment (DeKleineKobini) [inline extension/scripts/global/functions/containers.js:65]: `'` > escaping

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]

File: extension/scripts/features/abroad-people-filter/ttPeopleFilter.js (hunk 3 above)
Comment (Sashank999) [reply extension/scripts/features/abroad-people-filter/ttPeopleFilter.js:264]: Because `delete` takes 3ms while this does that fast.

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**